packages/venia-ui/lib/targets/__tests__/venia-ui-targets.spec.js

```diff
@@ -14,7 +14,7 @@ const thisDep = {
     intercept
 };
 
-const WEBPACK_BUILD_TIMEOUT = 20000;
+const WEBPACK_BUILD_TIMEOUT = 30000;
```

Comment:
It this related to story ?

Reply:
it is not, but it helps stabilize this test when run locally.